test/e2e/issues_test.go

```diff
@@ -1,6 +1,7 @@
 package e2e
```

Comment:
Do you mind updating some of the existing tests that do not leverage this feature to ensure the label/annotations are NOT created?